code/modules/mob/living/silicon/robot/death.dm

```diff
@@ -23,6 +26,8 @@
 	if(!gibbed)
 		logevent("FATAL -- SYSTEM HALT")
 		modularInterface.shutdown_computer()
+		internal = new /obj/item/gps/internal/cyborg(src)
+		internal.gpstag += " ([real_name])"
```

Comment:
This doesn't seem to turn off when the borg is revived

Reply:
Tested it ingame and it worked fine. GPS was successfully removed when I revived normally (cyborg emergency reboot module) and with an admin revive (Rejuvenate).